packages/nextjs/src/config/wrappers/withSentryGSProps.ts

```diff
@@ -0,0 +1,16 @@
+import { GSProps } from './types';
+import { callOriginal } from './wrapperUtils';
+
+/**
+ * Create a wrapped version of the user's exported `getStaticProps` function
+ *
+ * @param origGSProps: The user's `getStaticProps` function
+ * @returns A wrapped version of the function
+ */
+export function withSentryGSProps(origGSProps: GSProps['fn']): GSProps['wrappedFn'] {
```

Comment:
> difference between withSentryGSProps and withSentryGSSProps is a bit subtle

Agreed. That's why I went with `withSentryGSSP` for the latter, rather than `withSentryGSSProps`.

I don't feel strongly on this point. I can write them out if, given the above, it still feels unclear. They're only used internally, though honestly IDK if that's an argument for or against.